sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -983,6 +983,482 @@
+    {
+      "name": "spring.cloud.azure.keyvault.secret.client.headers[0].values",
+      "type": "java.util.List<java.lang.String>",
+      "description": "List of values of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.storage.blob.client.headers[0].name",
+      "type": "java.lang.String",
+      "description": "The name of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.storage.blob.client.headers[0].values",
+      "type": "java.util.List<java.lang.String>",
+      "description": "List of values of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.storage.fileshare.client.headers[0].name",
+      "type": "java.lang.String",
+      "description": "The name of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.storage.fileshare.client.headers[0].values",
+      "type": "java.util.List<java.lang.String>",
+      "description": "List of values of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.storage.queue.client.headers[0].name",
+      "type": "java.lang.String",
+      "description": "The name of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.storage.queue.client.headers[0].values",
+      "type": "java.util.List<java.lang.String>",
+      "description": "List of values of the header.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
+    },
+    {
+      "name": "spring.cloud.azure.keyvault.secret.property-sources[0].case-sensitive",
+      "type": "java.lang.Boolean",
+      "description": "Defines the constant for the property that enables/disables case-sensitive keys.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.keyvault.secrets.properties.AzureKeyVaultPropertySourceProperties"
+    },
+    {
+      "name": "spring.cloud.azure.keyvault.secret.property-sources[0].client.application-id",
+      "type": "java.lang.String",
+      "description": "Represents current application and is used for telemetry\/monitoring purposes.",
+      "sourceType": "com.azure.spring.cloud.autoconfigure.implementation.properties.core.client.HttpClientConfigurationProperties"
```

Comment:
It should be `com.azure.spring.cloud.autoconfigure.properties.core.client.ClientConfigurationProperties`

Reply:
updated